### PR TITLE
updated text to reflect current workflow practice

### PR DESCRIPTION
**Pull request by PietroLiuzzo**
he workflow and setup pages still referred to a direct workflow. I have also added a specific page on creation of records, which I think was missing.

### CODE REVIEW

File: pages/create.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+         <sourceDesc>
+            <p></p>
+         </sourceDesc>
+      </fileDesc>       <revisionDesc>
+        <change who="DR" when="2019-11-20">Created file</change></revisionDesc>
+   </teiHeader>
+   <text>
+      <body xml:id="create">
+<div type="level1">
+        <p>If you have editors rights, you can create a file once logged in the application from the relevant page.</p>
```
Comment (thea-m) [inline pages/create.xml]: I would add something like "Please follow the instructions given on the application after the creation of a new file. After downloading and before commiting the new file, always open it in your editor to make sure that it is valid and, if necessary (which it nearly always is!), edit it."

File: pages/create.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            <author>Pietro Liuzzo</author>
+         </titleStmt>
+         <publicationStmt>
+            <p/>
+          </publicationStmt>
+         <sourceDesc>
+            <p></p>
+         </sourceDesc>
+      </fileDesc>       <revisionDesc>
+        <change who="DR" when="2019-11-20">Created file</change></revisionDesc>
```
Comment (thea-m) [inline pages/create.xml]: I don't remember creating this file....

**Comment by PietroLiuzzo** [comment]
Just to mention it, you can switch to this branch and commit changes as well :)

**Comment by PietroLiuzzo** [comment]
I have been sooo smart to make the changes requested here in another branch, which I have now PRed ....

**Pull request merged**